plugins/out_datadog/datadog.c

```diff
@@ -321,7 +358,7 @@ static void cb_datadog_flush(struct flb_event_chunk *event_chunk,
     ret = datadog_format(config, i_ins,
                          ctx, NULL,
                          event_chunk->tag, flb_sds_len(event_chunk->tag),
-                         event_chunk->data, event_chunk->size,
+                         event_chunk->data, event_chunk->size, event_chunk->total_events,
```

Comment:
This change is causing the build to give a warning... will fix it... this is used in the `test_formatter.callback`